### PR TITLE
Link to dotnet

### PR DESCRIPTION
**Pull request by kviktest2**
Long description
How to install dotnet core.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# learning
\ No newline at end of file
+# learning
```
Comment (kviktest) [inline README.md:1]: Good idea, but make it better

**Comment by kviktest** [comment]
Thanks anyway :)

**Comment by kviktest2** [comment]
Your welcome :)

2017-12-14 21:01 GMT+01:00 kviktest <notifications@github.com>:

**Comment by kviktest** [comment]
+1

**Comment by kviktest2** [comment]
+1

**Pull request merged**